### PR TITLE
Altered installation guide to reflect Docker container

### PR DESCRIPTION
**Pull request by jamesvrt**
Before you submit a pull request, please fill in the following:

**Related Issue(s):**
#41 

**Description:**
Update the notebook template to better reflect working in a Docker container.

**PR checklist:**

- [X] Code is formatted (run `scripts/format`).
- [X] Code lints properly (run `scripts/lint`).
- [ ] Tests pass (run `scripts/test`).
- [X] Documentation has been updated to reflect changes, if applicable.
- [ ] Changes are added to the [CHANGELOG](../CHANGELOG.md).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -138,3 +138,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# VSCode
+.vscode/
```
Comment (cuttlefish) [inline .gitignore:143]: It's usually better to add this to your workstation's global `.gitignore`.
I think the default location is `$HOME/.config/git/` or you can set it with something like `git config --global core.excludesFile ~/.gitignore`

File: README.md [hunk 2]
```diff
@@ -13,6 +13,6 @@ This is a template repo used for creating new packages for `stactools`.
 3. Update `setup.cfg` with your package description and such.
 4. Update the LICENSE with your company's information (or whomever holds the copyright).
 5. Run `sphinx-quickstart` in the `docs` directory to create the documentation template.
-6. Update `docs/installation_and_basic_usage.ipynb` to provide an interactive notebook to help users get started. Include the following badge at the top of the README to launch the notebook: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stactools-packages/template/main?filepath=docs/installation_and_basic_usage.ipynb). Be sure to modify the badge href to match your package repo.
+6. Update `docs/installation_and_basic_usage.ipynb` to provide an interactive notebook to help users get started.
```
Comment (cuttlefish) [inline README.md:16]: What's the reason we're removing this?

File: docs/installation_and_basic_usage.ipynb [hunk 3]
```diff
@@ -2,294 +2,125 @@
...
    "source": [
-    "Notice the Commands available. In the next step we will add a dataset to work with"
-   ]
+    "## 1. Using this notebook\n",
+    "\n",
+    "The easiest way to use this notebook is to run it through `scripts/notebook`. This will create a Docker container already set up with the prerequisites and open the stactools dataset folder. Navigate to `docs/` to find this notebook.\n",
+    "\n",
+    "If you wish to use this notebook outside of the container then please install the prerequisites using:\n",
+    "\n",
+    "`pip install stac stactools-<name of this package>`"
```
Comment (cuttlefish) [inline docs/installation_and_basic_usage.ipynb:25]: If we want strings like this to be replaced during the `scripts/rename` we can change it to `pip install stactools-ephemeral`

File: README.md (hunk 2 above)
Comment (jamesvrt) [reply README.md:16]: My intention with these changes was to have the notebook run in the Docker container by default, so it wouldn't require any packages to be installed. I realised that meant the notebook wouldn't work on binder.org. Hindsight, leaving a few package install commands in the notebook to allow for greater portability (while still working in the container) was the better option.

File: docs/installation_and_basic_usage.ipynb [hunk 4]
```diff
@@ -2,294 +2,132 @@
...
-      "  merge           Merge items from one STAC into another.\n",
-      "  move-assets     Move or copy assets in a STAC to the Item locations.\n",
-      "  nrcanlandcover  Commands for working with Natural Resources Canada Land\n",
-      "                  Cover data\n",
-      "\n",
-      "  sentinel2       Commands for working with sentinel2 data\n",
-      "  version         Display version info.\n"
-     ]
-    }
+    "!pip install stactools stactools-ephemeral"
```
Comment (cuttlefish) [inline docs/installation_and_basic_usage.ipynb:31]: I think the `stactools` is redundant here?

**Review by cuttlefish — APPROVED** [review]

**Pull request merged**